test/e2e_node/cpu_manager_test.go

```diff
@@ -417,6 +436,64 @@ func runCPUManagerTests(f *framework.Framework) {
 
 		setOldKubeletConfig(f, oldCfg)
 	})
+
+	It("should reassigne CPUs after kubelet restart", func() {
+		oldCfg = enableCPUManagerInKubelet(f)
+
+		cpuCap, cpuAlloc, cpuRes = getLocalNodeCPUDetails(f)
+
+		// Skip CPU Manager tests if the number of allocatable CPUs < 1.
+		if cpuAlloc < 1 {
+			framework.Skipf("Skipping CPU Manager tests since the number of allocatable CPUs < 1")
+		}
+
+		By("running a Gu pod requesting multiple CPUs")
+		ctnAttrs = []ctnAttribute{
+			{
+				ctnName:    "gu-container",
+				cpuRequest: "2000m",
+				cpuLimit:   "2000m",
+			},
+		}
+		pod = makeCPUManagerPod("gu-pod", ctnAttrs, fmt.Sprintf("while :; do grep Cpus_allowed_list /proc/self/status | cut -f2 && sleep 0.5; done"))
+		pod = f.PodClient().CreateSync(pod)
+
+		By("checking if the expected cpuset was assigned")
+		cpuListString = "1-2"
+		if isHTEnabled() {
+			cpuListString = "2-3"
+			cpuList = cpuset.MustParse(getCPUSiblingList(0)).ToSlice()
+			if cpuList[1] != 1 {
+				cset = cpuset.MustParse(getCPUSiblingList(1))
+				cpuListString = fmt.Sprintf("%s", cset)
+			}
+		}
+
+		expAllowedCPUsListRegex = fmt.Sprintf("^%s$", cpuListString)
+		err = f.PodClient().MatchContainerLastLineOfOutput(pod.Name, pod.Spec.Containers[0].Name, expAllowedCPUsListRegex)
```

Comment:
Can you check if you can use https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/util.go#L2013? That way we don't need `MatchContainerLastLineOfOutput()`. It would also remove the necessity to pass the `cmd` in `makeCPUManagerPod()`.

Reply:
It doesn't work out of the box, such snippet:
```
func executeCmd(pod string) (string, error) {                                                                                                                                                  
    cmd := framework.KubectlCmd("exec", pod, "grep Cpus_allowed_list /proc/self/status | cut -f2")                                                                                             
    fmt.Printf("%v\n", cmd)                                                                                                                                                                    
    output, err := cmd.Output()                                                                                                                                                                    if err != nil {                                                                                                                                                                            
        return "", err                                                                                                                                                                         
    }                                                                                                                                                                                          
    return string(output), nil                                                                                                                                                                 }     
```

returns:
`&{ [ --server=http://127.0.0.1:8080 exec gu-pod grep Cpus_allowed_list /proc/self/status | cut -f2] []  <nil> <nil> <nil> [] <nil> <nil> <nil> <nil> <nil> false [] [] [] [] <nil> <nil>}`

so probably `TextContext.KubectlPath` is nil.
Look here:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/util.go#L2040